.pre-commit-config.yaml

```diff
@@ -24,10 +25,6 @@ repos:
   - id: blacken-docs
     additional_dependencies: [black]
     exclude: README.md
-- repo: https://github.com/pycqa/isort
-  rev: 5.12.0
-  hooks:
-  - id: isort
 - repo: https://github.com/pycqa/flake8
```

Comment:
We could prob replace `flake8` here with `ruff` too. I know `ruff` implements these plugins since I was using those as well.

```
    - pyproject-flake8==6.0.0
    - flake8-bugbear==22.12.6
    - flake8-typing-imports==1.14.0
    - flake8-docstrings==1.6.0
```

But haven't checked these two yet:

```
    - flake8-rst-docstrings==0.3.0
    - flake8-rst==0.8.0
```